core/src/config.cpp

```diff
@@ -42,6 +42,8 @@ const QStringList Config::cacheApiUrls = {
 
 const QString Config::apiUrlOverrideEnvironmentVariableName = "PK_LAUNCHER_API_URL";
 const QString Config::apiCacheUrlOverrideEnvironmentVariableName = "PK_LAUNCHER_API_CACHE_URL";
+const QString Config::appSecretOverrideEnvVar = "PK_LAUNCHER_FORCE_SECRET";
+const QString Config::appVersionIdOverrideEnvVar = "PK_LAUNCHER_FORCE_VERSION";
```

Comment:
I would rename it to
- `PK_LAUNCHER_FORCE_PATCHER_SECRET`
- `PK_LAUNCHER_FORCE_PATCHER_VERSION_ID`

This way it is 100% clear what we are changing.